indico/modules/rb_new/schemas.py

```diff
@@ -158,10 +158,12 @@ class BlockingSchema(mm.ModelSchema):
     blocked_rooms = Nested(BlockedRoomSchema, many=True)
     allowed = Nested(BlockingPrincipalSchema, many=True)
     can_edit = Function(lambda blocking: blocking.can_be_modified(session.user))
+    created_by = Function(lambda blocking: blocking.created_by_user.full_name)
```

Comment:
I think this would be cleaner in terms of semantics (untested!):

    created_by = Nested(UserSchema, attribute='created_by_user', only='full_name')